examples/update/step2/index.ts

```diff
@@ -8,3 +9,35 @@ const logGroup = new aws.s3.Bucket("bucket", {
         value: "buzz", // <-- this value has changed
     }]
 });
+
+const lambdaRole = new awsClassic.iam.Role("lambdaRole", {
+    assumeRolePolicy: awsClassic.iam.assumeRolePolicyForPrincipal({ Service: "lambda.amazonaws.com" }),
+    inlinePolicies: [{
+        name: "lambdaAccess", policy: JSON.stringify({
+            Version: "2012-10-17",
+            Statement: [{
+                Effect: "Allow",
+                Action: [
+                    "logs:CreateLogGroup",
+                    "logs:CreateLogStream",
+                    "logs:PutLogEvents",
+                ],
+                Resource: "arn:aws:logs:*:*:*",
+            }],
+        })
+    }]
+});
+
+const func = new aws.lambda.Function("function", {
+    runtime: "nodejs12.x",
```

Comment:
Might want to use a more recent runtime so we don't have to fix unexpectedly if they drop support. This is the oldest of the 3 supported node runtimes - 16.x is most recent.